server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1165,7 +1165,6 @@ public Node start() throws NodeValidationException {
             : "clusterService has a different local node than the factory provided";
         transportService.acceptIncomingRequests();
         extensionsOrchestrator.extensionsInitialize();
-        extensionsOrchestrator.setNamedWriteableRegistry();
```

Comment:
Why are we removing this?

Reply:
The actions it accomplishes are being moved into the initialize call immediately above it to remove the need for a public setter and more strongly enforce the correct sequencing.